include/wlc/wlc.h

```diff
@@ -280,6 +280,12 @@ bool wlc_output_get_sleep(wlc_handle output);
 /** Wake up / sleep. */
 void wlc_output_set_sleep(wlc_handle output, bool sleep);
 
+/** Set gamma */
+void wlc_output_set_gamma(wlc_handle output, uint16_t size, uint16_t *r, uint16_t *g, uint16_t *b);
```

Comment:
Yeah I dunno how gamma works, I just gave an interface on top of drm. Will mention that in the docs.